src/view/Office.vue

```diff
@@ -22,9 +22,27 @@
 
 <template>
 	<transition name="fade" appear>
-		<div v-show="!loading" id="richdocuments-wrapper">
-			<div class="header">
-				<!-- This is obviously not the way to go since it would require absolute positioning and therefore not be compatible with viewer actions/sidebar -->
+		<div id="richdocuments-wrapper">
+			<div v-if="showLoadingIndicator" id="cool-loading-overlay" :class="{ debug: debug }">
```

Comment:
Small detail but: Isn't it more readable when each attribute is on a new line?

Reply:
Up to 3 are allowed for a single line, but agreed that it is more readable. I'll split that.